tests/unicode/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -69,3 +69,12 @@ function utf16Length(str, pos) {
 }
 "
 `;
+
+exports[`test keys.js 1`] = `
+"({\'この事はつもり素晴らしいことさ\': \'35jL9V\'})
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+({
+  \"\\u3053\\u306E\\u4E8B\\u306F\\u3064\\u3082\\u308A\\u7D20\\u6674\\u3089\\u3057\\u3044\\u3053\\u3068\\u3055\": \"35jL9V\"
```

Comment:
There is a prettier issue somewhere about this. Right now we aggressively escape everything in order to be ascii compatible. We'll probably want to evaluate what we want to do long term.